docs/FeatureManagement/Samples/NoFilters.sample.json

```diff
@@ -0,0 +1,60 @@
+{
+    "feature_management": {
+        "feature_flags": [
+            {
+                "id": "BooleanTrue",
+                "description": "An Enabled Feature Flag with no Filters.",
+                "enabled": true,
+                "conditions": {
+                    "client_filters": []
+                }
+            },
+            {
+                "id": "BooleanFalse",
+                "description": "A Disabled Feature Flag with no Filters.",
+                "enabled": false,
+                "conditions": {
+                    "client_filters": []
+                }
+            },
+            {
+                "id": "StringTrue",
+                "description": "An Enabled Feature Flag with no Filters.",
+                "enabled": "true",
+                "conditions": {
+                    "client_filters": []
+                }
+            },
+            {
+                "id": "StringFalse",
+                "description": "A Disabled Feature Flag with no Filters.",
+                "enabled": "false",
+                "conditions": {
+                    "client_filters": []
+                }
+            },
+            {
+                "id": "InvalidEnabled",
+                "description": "A Feature Flag with an invalid 'enabled' Value.",
+                "enabled": "invalid",
+                "conditions": {
+                    "client_filters": []
+                }
+            },
+            {
+                "id": "Minimal",
+                "enabled": true
+            },
+            {
+                "id": "NoEnabled"
+            },
+            {
+                "id": "EmptyConditions",
+                "description": "A Disabled Feature Flag with no Conditions.",
```

Comment:
Better to use "empty Conditions" instead of "no Conditions".
"no Conditions" sounds like there is no `conditions` property.

Reply:
It's a little confusing by saying "a disabled feature flag" while in this sample it's `enabled: true`.  Maybe something like "A disabled feature flag **due to** empty conditions."?

BTW, there is chaos about capitalization, is that on purpose?

Reply:
I'm also confused with the capitalization.

Reply:
Why is this a "disabled" feature flag?

Reply:
Never mind just took a look at it. It should say enabled.